.github/workflows/python-tests.yml

```diff
@@ -22,13 +22,15 @@ jobs:
       matrix:
         python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
         os: [macos-latest, ubuntu-latest, windows-latest]
-#       exclude:
-#         - os: windows-latest
-#           python-version: '3.7'
+        exclude:
+          - os: windows-latest
+            python-version: '3.6'
+          - os: windows-latest
+            python-version: '3.7'
```

Comment:
Can you combine these values as above? i.e. `python-version: ['3.6', '3.7']`

Reply:
This was my 1st trial ! It does not work, both tests were still active.